test/spec/ol/style/style.test.js

```diff
@@ -91,12 +102,16 @@ describe('ol.style.Style', function () {
       expect(original.getGeometry()).not.to.be(clone.getGeometry());
       expect(original.getFill()).not.to.be(clone.getFill());
       expect(original.getImage()).not.to.be(clone.getImage());
+      expect(original.getRenderer()).not.to.be(clone.getRenderer());
```

Comment:
This test currently fails, since both the original and the clone reference the same renderer function.  This is expected in the absence of a deep clone, as was discussed in the issue #11860 thread.  Should I change line 105 to `expect(original.getRenderer()).to.be(clone.getRenderer());`, or delete that test entirely?

Reply:
I would remove this line.